easybuild/easyblocks/n/netcdf.py

```diff
@@ -69,18 +69,22 @@ def configure_step(self):
 
         else:
             self.cfg.update('configopts', '-DCMAKE_BUILD_TYPE=RELEASE -DCMAKE_C_FLAGS_RELEASE="-DNDEBUG " ')
```

Comment:
While you're working on it, this line is wrong.
It should not set this if toolchainopt debug= True but rather BUILD_TYPE=Debug and so on...

Reply:
Added a conditional on `toolchain.options['debug']`. CMakeLists already adds flags for debug builds, so I removed any further settings.